src/main/java/frc/robot/constants/fields/CompetitionField.java

```diff
@@ -0,0 +1,18 @@
+package frc.robot.constants.fields;
+
+import edu.wpi.first.wpilibj.geometry.Translation2d;
+import frc.robot.constants.FieldConstants;
+
+/**
+ * Competition field way-points. relative to the (0,0) point in PathWeaver. all
+ * measurement are measured in meters.
+ */
+public class CompetitionField extends FieldConstants {
+    // TODO: check reality
+    public CompetitionField() {
+        TRENCH_RUN_ENTRANCE = new Translation2d(5.45, -0.70485);
+        MIDDLE_OF_TRENCH = new Translation2d(9.294, -0.7112);
+        FRONT_RIGHT_BALL = new Translation2d(6.193, -4.478);
+        FRONT_LEFT_BALL = new Translation2d(5.902, -3.656);
```

Comment:
Just asking, did you measure them in pathweaver or in the manual?

Reply:
most using pathweaver